### ISSUE TITLE
Failed to install: cordova-plugin-mauron85-background-geolocation

### ISSUE THREAD

**Issue by shyamal890** [mauron85/cordova-plugin-background-geolocation#561]
Trying to install with `cordova plugin add @mauron85/cordova-plugin-background-geolocation` and it gives the following error:

```
Error during processing of action! Attempting to revert...
Failed to install 'cordova-plugin-mauron85-background-geolocation': CordovaError: Uh oh!
"E:\Dropbox\SmartTask Mob\SmartTask.Mob4\plugins\cordova-plugin-mauron85-background-geolocation\android\common\src\main\java\com\evgenii\jsevaluator\HandlerWrapper.java" not found!
    at copyFile (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\lib\pluginHandlers.js:233:36)
    at copyNewFile (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\lib\pluginHandlers.js:261:5)
    at install (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\lib\pluginHandlers.js:43:17)
    at ActionStack.process (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\node_modules\cordova-common\src\ActionStack.js:56:25)
    at PluginManager.doOperation (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:114:20)
    at PluginManager.addPlugin (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\node_modules\cordova-common\src\PluginManager.js:144:17)
    at E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\Api.js:247:74
    at _fulfilled (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\node_modules\q\q.js:834:54)
    at E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\node_modules\q\q.js:863:30
Uh oh!
"E:\Dropbox\SmartTask Mob\SmartTask.Mob4\plugins\cordova-plugin-mauron85-background-geolocation\android\common\src\main\java\com\evgenii\jsevaluator\HandlerWrapper.java" not found!
    at Promise.promise.promiseDispatch (E:\Dropbox\SmartTask Mob\SmartTask.Mob4\platforms\android\cordova\node_modules\q\q.js:796:13)
[ERROR] An error occurred while running subprocess cordova.

        cordova plugin add @mauron85/cordova-plugin-background-geolocation --save exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.
```

**Comment by mauron85** [comment]
Make sure you've removed all traces of previous pkg. The best you can do is remove platforms and plugins dir and node_modules and start fresh.

**Comment by shyamal890** [comment]
I tried removing platforms, plugins and node_modules but it still didn't work.

**Comment by mauron85** [comment]
You're right. My bad forget to put submodules into npm pkg. Should fixed in 3.03.

```
cordova plugin add @mauron85/cordova-plugin-background-geolocation@3.0.3
```

**Comment by mauron85** [comment]
Should be resolved by now. Closing
